.github/workflows/ci.yml

```diff
@@ -0,0 +1,19 @@
+name: C/C++ CI
+
+on: [push, pull_request]
+
+jobs:
+  build:
+    name: ubuntu-latest
+    runs-on: ubuntu-latest
+    container: devkitpro/devkitppc:latest
+ 
+    steps:
+    - uses: actions/checkout@v3.5.2
+
+    - name: Build
+      run: |
+        make -j2
+    
+    - name: Upload  Build Artifacts
```

Comment:
uploading artifacts from an examples repository just seems like a waste of resources to me.